src/XMLimport.cpp

```diff
@@ -966,8 +966,9 @@ void XMLimport::readHostPackage(Host* pHost)
         readNext();
 
         if (isEndElement()) {
-            break;
-        } else if (isStartElement()) {
+            continue;
```

Comment:
This change is the problem.

Reply:
Hmm I had strange issues where without this change nothing else was loaded.... maybe something went wrong during compilation.

Reply:
I'll revert that part and check.

Reply:
This is awkward... Now it works like it should with break. Sorry for confusion, changes is reverted.

Reply:
Should be ok.